ui/src/gear/groups/lib.ts

```diff
@@ -0,0 +1,227 @@
+import { deSig } from '@urbit/aura';
```

Comment:
All these groups files need to match the latest from groups this is all old stuff. honestly I think it's ok to remove them for now since it'll be a lot to come up with these for latest groups.